Doc/library/compileall.rst

```diff
@@ -113,6 +113,11 @@ compile Python sources.
 
    Ignore symlinks pointing outside the given directory.
 
+.. cmdoption:: --hardlink-dupes
+
+   Use hardlinks to prevent duplicates if ``.pyc`` files for multiple
+   optimization levels have the same content.
```

Comment:
Does this not need a versionchanged/versionadded note?

Reply:
It is mentioned a few lines below in `.. versionchanged:: 3.9` section.

Reply:
ok